freshmaker/errata.py

```diff
@@ -318,7 +318,7 @@ def _rhel_release_from_product_version(self, errata_id, product_version):
 
         return data["rhel_release"]["name"]
 
-    def get_builds(self, errata_id, rhel_release_prefix=None):
+    def get_builds(self, errata_id, rhel_release_prefix=None, return_srpm=True):
```

Comment:
btw, this api has a bug that module nvr is returned in the nvr list when we're getting the srpm nvrs, could you fix it as well?

Reply:
What exactly do you mean by **module nvr**?  For example for that advisory builds https://errata.devel.redhat.com/api/v1/erratum/53134/builds.json you will have **nodejs-10-8010020200318152252.c27ad7f8** in the nvrs list?

Reply:
Yes, that's what I mean, *nodejs-10-8010020200318152252.c27ad7f8* never appears in a systems (s)rpm list, so it should be excluded from the srpm nvrs. And I believe we can simplify the code by removing `get_srpm_nvrs` and `get_binary_nvrs`, only get source rpm nvrs and rpm nvrs from the actual rpms which is under the arches, so it's unnecessary have the code to get the build nvrs.